agent/lib/kontena/workers/stats_worker.rb

```diff
@@ -13,7 +13,7 @@ def initialize(queue, autostart = true)
       @queue = queue
       @statsd = nil
       @node_name = nil
-      info 'initialized'
+      debug 'initialized'
```

Comment:
This should stay as `info`, it only happens once at startup and then again after a worker crash.